src/styles/settings.scss

```diff
@@ -1,31 +1,75 @@
-$base-padding: 24px;
+// WE DON'T USE THESE VARIABLES IN ANY OTHER FILE
 
-$header-topbar-bg: #434343;
-$header-bg: #363636;
+$darkGray: #2a2a2a;
+$veryDarkGray: #292929;
+$mediumDarkGray: #b6b6b6;
+$gray: #a5a5a5;
+$mediumGray: #a9a9a9;
+$lightGray: #d8d8d8;
+$mediumLightGray: #e2e2e2;
+$veryLightGray: #e1e1e1;
+$lightGrayishBlue: #e0e3ed;
+$mediumLightGrayishBlue: #f4f5f7;
+$battleshipGray: #434343;
+$shadow: #363636;
+$orange: #d58e32;
+$white: #ffffff;
 
-$footer-claim-bg: #2a2a2a;
-$footer-bg: #363636;
+$header-topbar-bg: $battleshipGray;
+$header-topbar-a: $white;
+$header-bg: $shadow;
 
-$primary: #d58e32;
+$newFurniture-panelBar-border: $mediumLightGray;
+$newFurniture-menu-a: $darkGray;
+$newFurniture-dots-bgc: $white;
+$newFurniture-dots-a: $veryLightGray;
 
-$text-color: #2a2a2a;
+$companyClaim-phoneNumber-p: $lightGray;
+$companyClaim-cardCounter-color: $lightGrayishBlue;
+$companyClaim-cardBox-color: $white;
 
-$sale-color: #ffff;
+$menuBar-active-color: $white;
+$menuBar-a-border: $white;
+$menuBar-root-bgc: $white;
 
-$text-color-two: rgb(169, 169, 169);
+$footer-claim-bg: $darkGray;
+$footer-bg: $shadow;
+$footer-menu-h6: $white;
+$footer-menu-a: $mediumGray;
+$footer-menu-hover: $white;
+$footer-copyright-p: $mediumGray;
+$footer-socialMedia-a: $mediumGray;
 
-$form-border-color: #292929;
+$productBox-root-bgc: $white;
+$productBox-root-border: $veryLightGray;
+$productBox-photo-color: $mediumLightGrayishBlue;
+$productBox-sale-bgc: $veryDarkGray;
+$productBox-sale-color: $white;
+$productBox-stars-color: $darkGray;
+$productBox-line-border: $darkGray;
 
-$line-color: #000;
+$featureBox-root-border: $mediumDarkGray;
+$featureBox-root-icon-color: $white;
+$featureBox-icon-border: $mediumDarkGray;
+$featureBox-icon-bgc: $white;
+$featureBox-content-color: $darkGray;
 
-$old-price: #ff0000;
+$button-main-bgc: $darkGray;
+$button-main-color: $white;
+$button-main-not-color: $white;
+$button-main-not-color: $white;
+$button-outline-border: $darkGray;
+$button-outline-color: $darkGray;
+$button-outline-not-bgc: $darkGray;
+$button-outline-not-color: $white;
+$button-favorite-color: $white;
+$button-compare-color: $white;
 
-$favourite-color: #ff0000;
+$primary: $orange;
 
-$compare-color: #228b22;
+$text-color: $darkGray;
+$old-price-text-color: $gray;
 
-$transition: all 0.25s;
+$form-border-color: $veryDarkGray;
 
-%hover-effect {
-  transition: $transition;
-}
```

Comment:
Dlaczego usunęłaś ten kod - to nie ma zwiążku z tym zadaniem? Przywróć go rposzę

Reply:
przywroce